shell/src/main/java/alluxio/cli/LogLevel.java

```diff
@@ -56,15 +59,17 @@
 public final class LogLevel {
   private static final Logger LOG = LoggerFactory.getLogger(LogLevel.class);
 
-  private static final String LOG_LEVEL = "logLevel";
-  private static final String ROLE_WORKERS = "workers";
-  private static final String ROLE_MASTER = "master";
-  private static final String ROLE_WORKER = "worker";
-  private static final String ROLE_JOB_MASTER = "job_master";
-  private static final String ROLE_JOB_WORKER = "job_worker";
-  private static final String ROLE_JOB_WORKERS = "job_workers";
-  private static final String TARGET_SEPARATOR = ",";
-  private static final String TARGET_OPTION_NAME = "target";
+  public static final String LOG_LEVEL = "logLevel";
+  public static final String ROLE_WORKERS = "workers";
+  public static final String ROLE_MASTER = "master";
+  public static final String ROLE_MASTERS = "masters";
+  public static final String ROLE_WORKER = "worker";
+  public static final String ROLE_JOB_MASTER = "job_master";
+  public static final String ROLE_JOB_MASTERS = "job_masters";
```

Comment:
`masters` and `job_masters` are not valid options. Instead of rejecting them, we give a warning and translate it to the correct one.